docs/source/diagrams/ReflectometryReductionOne_TransmissionCorrection-v1_wkflw.dot

```diff
@@ -0,0 +1,59 @@
+digraph ReflectometryReductionOne {
+label = "ReflectometryReductionOne Transmission Correction Flowchart"
+ $global_style
+
+subgraph params {
+ $param_style
+  inputWorkspace    [label="InputWorkspace"]
+  outputWorkspace   [label="OutputWorkspace"]
+  corrAlg           [label="CorrectionAlgorithm"]
+  firstTransRun     [label="FirstTransmissionRun"]
+  secondTransRun    [label="SecondTransmissionRun"]
+}
+
+subgraph decisions {
+ $decision_style
+  checkTransRun   [label="FirstTransmissionRun\ngiven?"]
+  checkCorrAlg    [label="CorrectionAlgorithm?"]
+  checkTransRun   [label="FirstTransmissionRun\ngiven?"]
+  checkTransUnits [label="X axis in &lambda;?"]
+}
+
+subgraph algorithms {
+ $algorithm_style
+  rebinByTransWS  [label="RebinToWorkspace"]
+  polyCorr        [label="PolynomialCorrection"]
+  expCorr         [label="ExponentialCorrection"]
+  createTransWS   [label="CreateTransmissionWorkspace"]
+  rebinByTransWS  [label="RebinToWorkspace"]
+  divideTrans     [label="Divide\n(InputWorkspace / TransmissionWorkspace)"]
+}
+
+subgraph processes {
+ $process_style
+}
+
+subgraph values {
+ $value_style
+  valTrans        [label="It(&lambda;) / &Sigma;It(&lambda;)"]
+}
+
+inputWorkspace    -> checkTransRun
```

Comment:
We must be careful with using `InputWorkspace` here (the same applies to subsequent diagrams). Strictly speaking this is not our `InputWorkspace` but our `DetectorWorkspace` after the normalization. I don't mind keeping this name as long as we add a sentence somewhere in this section stating that this is the `InputWorkspace` once it has converted to wavelength and normalized by monitors.